File: Python/Source/Histogram.py
class Histogram:

    @staticmethod
    def hist(s):
        count = {}
        for letter in s:
            count[letter] = letter in count and count[letter] + 1 or 1

        res = []
        for letter in 'uwxz':
            if letter in count:
                string = '{letter}  {num:<5} {star:*<{num}}'.format(letter=letter, num=count[letter], star='*')
                res.append(string)

        return '\n'.join(res)

File: Go/Source/Histogram.go
package main

import (
	"fmt"
	"strings"
)

type Histogram struct{}

func (h Histogram) Hist(s string) string
{
	count := make(map[rune]int)
	for _, letter := range s
	{
		count[letter]++
	}

	var res []string
	for _, letter := range "uwxz"
	{
		if count[letter] > 0
		{
			num := fmt.Sprintf("%-6d", count[letter])
			stars := strings.Repeat("*", count[letter])
			res = append(res, fmt.Sprintf("%c  %s%s", letter, num, stars))
		}
	}
	return strings.Join(res, "\n")
}
